spec/models/solidus_bolt/bolt_configuration_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+
+RSpec.describe SolidusBolt::BoltConfiguration, type: :model do
+  let(:column_list) {
+    [
+      'id',
+      'bearer_token',
+      'environment_url',
+      'merchant_public_id',
+      'merchant_id',
+      'api_key',
+      'signing_secret',
+      'publishable_key',
+      'created_at',
+      'updated_at'
+    ]
+  }
+
+  describe 'Check Model Integrity' do
+    it 'has correct table name' do
+      expect(described_class.table_name).to eq 'solidus_bolt_bolt_configurations'
+    end
+
+    it 'has correct attributes' do
+      expect(described_class.new.attributes.keys.sort).to eq column_list.sort
+    end
+  end
+
+  describe '#fetch' do
+    it 'fetches the correct record' do
+      bolt_configuration = create(:bolt_configuration)
```

Comment:
You have to add another spec to test that the record is created when is not present.

Reply:
Added a test to make sure `first_or_create` is working
check here [365a6d4](https://github.com/nebulab/solidus_bolt/commit/365a6d43e7e8a049df560bde89c04183e14bda7a)